include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -3265,12 +3270,14 @@ namespace internal
   template <int dim, int fe_degree, typename Number>
   struct CellwiseInverseMassMatrixImpl
   {
-    template <typename FEEvaluationType>
     static void
-    apply(const unsigned int      n_components,
-          const FEEvaluationType &fe_eval,
-          const Number *          in_array,
-          Number *                out_array)
+    apply(const unsigned int                        n_components,
+          const FEEvaluationMappingRelated<dim,
+                                           typename Number::value_type,
+                                           false,
+                                           Number> &fe_eval,
+          const Number *                            in_array,
+          Number *                                  out_array)
```

Comment:
This function is one of the intended side effects of the proposed change: We can now compute the inverse mass matrix operation without a template parameter on the number of components.